eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -12,15 +12,13 @@ parameters:
 #   "TenantId": "<tenant id>",
 #   "TestApplicationId": "<test app id>",
 #   "TestApplicationSecret": "<test app secret>",
-#   "ProvisionerApplicationId": "<provisioner app id>",
-#   "ProvisionerApplicationSecret": "<provisioner app secret>",
+#   "ProvisionerApplicationId": "<provisoner app id>",
```

Comment:
Everything was spelled right before. And you're not consistent here or below. "secert" is also spelled wrong.